src/components/dashboard/DashboardTable.vue

```diff
@@ -84,8 +86,10 @@ export default {
         {label: 'Date', field: 'dateCreated'},
         {label: 'Submitter', field: 'submitter'},
         {label: 'Workflow Name', field: 'workflowName'},
-        {label: 'Source Item', field: 'itemName'},
-        {label: 'Source Filename', field: 'primaryfileName'},
+        {label: 'Collection', field: 'collectionName'},
+        {label: 'External Id', field: 'externalId'},
+        {label: 'Item', field: 'itemName'},
+        {label: 'Filename', field: 'primaryfileName'},
         {label: 'Workflow Step', field: 'workflowStep'},
```

Comment:
Let's call it Primary File instead of Filename, not to be confused with the media filename.

Reply:
Do we have room for external source? if not that's ok